pkg/apiserver/certificate/cacert_controller.go

```diff
@@ -130,40 +131,46 @@ func (c *CACertController) syncCACert() error {
 // syncMutatingWebhooks updates the CABundle of the MutatingWebhookConfiguration backed by antrea-controller.
 func (c *CACertController) syncMutatingWebhooks(caCert []byte) error {
 	klog.Info("Syncing CA certificate with MutatingWebhookConfigurations")
-	for _, name := range c.caConfig.MutationWebhooks {
-		mWebhook, err := c.client.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(context.TODO(), name, metav1.GetOptions{})
-		if err != nil {
-			return fmt.Errorf("error getting MutatingWebhookConfiguration %s: %v", name, err)
-		}
-		err = c.patchWebhookWithCACert(mWebhook, caCert)
-		if err != nil {
-			return fmt.Errorf("error updating antrea CA cert of MutatingWebhookConfiguration %s: %v", name, err)
-		}
+	if c.caConfig.MutationWebhookSelector == nil {
+		return nil
 	}
-	for _, name := range c.caConfig.OptionalMutationWebhooks {
-		mWebhook, err := c.client.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(context.TODO(), name, metav1.GetOptions{})
-		if err != nil {
-			if errors.IsNotFound(err) {
-				klog.V(2).Infof("Optional mutation webhook %s not found, skipping its update", name)
-				continue
-			}
-			return fmt.Errorf("error getting MutatingWebhookConfiguration %s: %v", name, err)
-		}
-		err = c.patchWebhookWithCACert(mWebhook, caCert)
+	listOption := metav1.ListOptions{LabelSelector: getLabelString(c.caConfig.MutationWebhookSelector)}
+	mWebhooks, err := c.client.AdmissionregistrationV1().MutatingWebhookConfigurations().List(context.TODO(), listOption)
+	if err != nil {
+		return fmt.Errorf("error listing Antrea MutatingWebhookConfiguration: %v", err)
+	}
+	for i := range mWebhooks.Items {
+		mWebhook := mWebhooks.Items[i]
+		name := mWebhook.Name
+		err = c.patchWebhookWithCACert(&mWebhook, caCert)
 		if err != nil {
-			return fmt.Errorf("error updating antrea CA cert of MutatingWebhookConfiguration %s: %v", name, err)
+			return fmt.Errorf("error updating Antrea CA cert of MutatingWebhookConfiguration %s: %v", name, err)
 		}
 	}
 	return nil
 }
 
+func getLabelString(selector *metav1.LabelSelector) string {
+	labelStrs := []string{}
+	for k, v := range selector.MatchLabels {
+		labelStrs = append(labelStrs, fmt.Sprintf("%s=%s", k, v))
+	}
+	return strings.Join(labelStrs, ",")
+}
```

Comment:
can be substituted by `metav1.FormatLabelSelector`

Reply:
done.